server/chat-commands.js

```diff
@@ -419,6 +419,27 @@ const commands = {
 		return target;
 	},
 
+	'!shrug': true,
+	shrug() {
+		if (!this.canTalk()) return;
+		return '¯\\_(ツ)_/¯';
```

Comment:
You don't need the double-backslash, PS, unlike Discord, is actually designed _not_ to mangle kaomoji.

Reply:
JS tried escaping the `_` when I tested it

Reply:
This is because JavaScript

Reply:
Ohhhhh lol nvm